platform/server/src/docker/config.yml

```diff
@@ -31,52 +31,54 @@ logging:
     io.dropwizard: INFO
 
 # This may refer to a remote Triplestore, e.g. https://example.org/sparql
-resources: /opt/trellis/data/rdf
+resources: ${TRELLIS_RESOURCES:-/opt/trellis/data/rdf}
```

Comment:
It seems like using relative paths in the defaults here would prevent a user having to update many values (or supply many ENV vars).

Reply:
I tend to prefer absolute paths in configuration files because it eliminates the ambiguity of the execution context.

Reply:
That ambiguity is only avoided in case where everything is installed in the same place anyway.  My premise is that an alternate location (not `/opt/trellis`) is used.  In that case, the clarity is a burdensome incorrectness.

Reply:
Admittedly, my perspective is coming from the container case, where the execution is highly defined and constrained.  It would be different for an app that might be started up different ways by an admin on CLI.

Reply:
The containerized use case is certainly very important, but Trellis is fundamentally a framework, not a packaged, ready-to-run application.